.docs/content/guide/2.tests.md

```diff
@@ -0,0 +1,52 @@
+# Run tests
+
+## How to use the ArmoniK C++ SDK
+
+### Build worker and dynamic worker (Linux only)
+
+To use ArmoniK with the C++ SDK, a dynamic worker image must be built. This worker image is then deployed with the other ArmoniK components during the deployment phase. The worker runtime provides to ArmoniK clients a set of business services. The worker must be configured to load a shared library containing the implementation of the calculation library related to the provided services.
+
+In order to build the dynamic worker image (```armonik-sdk-cpp-dynamicworker```) and the worker test (```libArmoniK.SDK.Worker.Test.so```), please run the following script:
+
+```bash
+./build-deploy-end2end.sh
+```
+
+This script generates the dynamic worker image that is used to deploy ArmoniK with. The script also generates the shared library used by the worker to perform the ArmoniK call on the desired services.  The script will also copy this shared library into the right folder so it can be loaded by the ArmoniK worker.
```

Comment:
```suggestion
This script generates the dynamic worker image that is used to deploy ArmoniK with. The script also generates the shared library used by the worker to perform the ArmoniK call on the desired services. The script will also copy this shared library into the right folder so it can be loaded by the ArmoniK worker.
```